client/my-sites/plans-grid/components/actions.tsx

```diff
@@ -491,6 +501,9 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 				busy={
 					isFreePlan( planSlug ) && planActionOverrides?.loggedInFreePlan?.status === 'blocked'
 				}
+				postButtonText={
+					isBusinessPlan( planSlug ) && planActionOverrides?.trialAlreadyUsed?.postButtonText
```

Comment:
I think may be better to pass `planActionOverrides` and do the conditioning in `SignupFlowPlanFeatureActionButton`? WDYT

and `busy` can be refactored later similarly. 
I'm also thinking `planActionOverrides` would be suited for a context prop

Reply:
> You mean, by adding it [inside this provider](https://github.com/Automattic/wp-calypso/blob/add/not-eligible-message-to-plans-selection/client/my-sites/plans-grid/grid-context.tsx#L26)?

Yes. We've been using that to reduce prop drilling for some things. It might be the case we migrate all of those props into the `wpcom-plans-ui` data-store though (we have it already and use it for a couple of cases).